MIDIchlorians/MIDIchlorians/AnimationTableViewController.swift

```diff
@@ -48,6 +48,7 @@ class AnimationTableViewController: UITableViewController {
         tableView.separatorStyle = .none
         tableView.separatorColor = Config.TableViewSeparatorColor
         tableView.register(AnimationTableViewCell.self, forCellReuseIdentifier: reuseIdentifier)
+        tableView.accessibilityLabel = "Animation Table"
```

Comment:
I know you are using these labels to refer to them from the tests, but are they considered magic strings?